packages/sw-background-sync-queue/src/lib/response-manager.js

```diff
@@ -11,6 +16,12 @@ async function putResponse({hash, idbObject, response, idbQDb}) {
 	_idbQHelper.put(hash, idbObject);
 }
 
+/**
+ * response fetched for the request for given hash
+ *
+ * @param {String} hash
+ * @return {Object} response
+ */
 async function getResponse(hash) {
```

Comment:
Can you wrap `hash` in an Object parameter, too? I know it's not as useful when there's only one parameter, but it's more confusing to have some methods take Objects and some take position parameters, and it also makes it easier to add in additional parameters later on.